readers/src/main/scala/com/salesforce/op/readers/Reader.scala

```diff
@@ -66,7 +66,8 @@ private[readers] trait ReaderType[T] extends Serializable {
    * @param opParams contains map of reader type to ReaderParams instances
    * @return ReaderParams instance if it exists
    */
-  final def getReaderParams(opParams: OpParams): Option[ReaderParams] = opParams.readerParams.get(this.typeName)
+  final def getReaderParams(opParams: OpParams): Option[ReaderParams] = opParams.readerParams
+    .get(this.wtt.tpe.toString.split("\\.").last)
```

Comment:
This seems invalid, since it will only use the type name without package prefix.

Reply:
Yes that's what we want. Example of reader param is `Map{"entity1" -> ...}`

Reply:
well it does match the check in the `readerInputTypes` then ;) I propose to use the full aliased name then `tpe.toString`

Reply:
That would involve changing all our OpParams...